search/mongo_search.go

```diff
@@ -14,6 +14,8 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+var opInterruptedCode = 11601
```

Comment:
Regarding the magic number, how/where did you find it in the first place?

Regarding regarding the well-known approach, it's fine if it's not in official MongoDB documentation -- I just wanted our code to show that the community accepts this approach (it wasn't something we just thought up and did on our own).